data-raw/overwrite_demo.R

```diff
@@ -2,4 +2,4 @@
 path <- here::here("data-raw/overwrite.csv")
 overwrite_demo <- readr::read_csv(path)
 
-usethis::use_data(overwrite_demo, overwrite = FALSE)
+usethis::use_data(overwrite_demo, overwrite = TRUE)
```

Comment:
Sorry, my commit message was confusing. Where I wrote `FALSE` I meant to write `TRUE`.